test/integration/controlplane/synthetic_controlplane_test.go

```diff
@@ -325,13 +364,13 @@ func TestObjectSizeResponses(t *testing.T) {
 	expectedMsgFor3MB := `Request entity too large: limit is 3145728`
 	expectedMsgForLargeAnnotation := `metadata.annotations: Too long: must have at most 262144 bytes`
 
-	deployment1 := constructBody("a", DeploymentMegabyteSize, "labels", t)      // >1 MB file
+	deployment1 := constructBody("a", DeploymentMegabyteSize, "labels", t)      // >1.5 MB file
```

Comment:
what has changed here

Reply:
how are the maths here?

 "a" is 4 bytes , it is a rune, then this happens
```go
	case "labels":
		labelsMap := map[string]string{}
		for i := 0; i < size; i++ {
			key := val + strconv.Itoa(i)
			labelsMap[key] = val
		}
		deploymentObject.ObjectMeta.Labels = labelsMap
```

key is 8 bytes and val 4 bytes , and this by DeploymentMegabyteSize  ... 12 x 130000 = 1560000
 
> 1.5MB , cool